Source/Scene/Globe.js

```diff
@@ -188,6 +188,22 @@ define([
          */
         this.depthTestAgainstTerrain = false;
 
+        /**
+         * Determines whether the globe will cast shadows when shadow mapping is enabled.
```

Comment:
Can we beef up this description (and to a lesser extent, the description for `receiveShadows`).
* Be explicit that shadows are cast onto anything with `receiveShadows` set to `true`, not just terrain, and vice versa - shadows can be received by other things.
* This can impact performance since the terrain is rendered a second time from the perspective of the sun/light, which will also load terrain tiles potentially out-side of view.